### DISCUSSION TITLE
Nightly vs Master versions

### DISCUSSION THREAD

**Discussion by samudiogo** [OrchardCMS/OrchardCore#7113] [General]
Hi guys, there is a considerable gaping between the nightly version and the master. I took a look at our milestones and roadmap, but couldn't find a date. A member of my team is using rc2-14258 and the latest (at this moment) is rc2-14414!

Is there any perspective to update our master branch in this season?

Thanks

**Comment by hishamco** [comment]
Hi @samudiogo the `dev` branch has the nightly builds which increments frequently, once we ship a release we update the `master` branch and publish the packages into NuGet.org
